reVX/plexos/plexos_plants.py

```diff
@@ -354,7 +354,7 @@ def _haversine_dist(cls, plant_coords, sc_coords):
         if plant_coords.shape[0] == 1:
             dist = dist.flatten()
 
-        R = 6373.0  # radius of the earth in kilometers
+        R = 6371.0  # radius of the earth in kilometers
```

Comment:
Did the earth shrink?

Reply:
I for the life of me could not figure out where you got 6373! Everything i found online said 6371, we should get consensus on that...

Reply:
Fortunately i think this is just a scalar on the distance so the relative and nearest distances should be all good. The radius depends on where you're measuring since the earth is not a perfect sphere. seems to be 6378 at the equator. But yes i'm also seeing 6371 as the average. Might have been a typo....